torch/_dynamo/variables/user_defined.py

```diff
@@ -65,7 +65,7 @@ def python_type(self):
     def as_proxy(self):
         return self.value
 
-    def __repr__(self):
+    def __str__(self):
```

Comment:
We define ```__repr__``` for other variables, I'm curious if this is intended or just either way is fine.

Reply:
Intended, if you print one of these with: `f"{var}"` the `__str__` in the base class is called and `__repr__` is not used.